List.css

```diff
@@ -0,0 +1,89 @@
+  .list-container 
+  {  
+    align-items: center;
+    background-color: var(--component-container-color);
+    border-radius: 0.3rem;
+    display: flex;
+    font-size: .9rem;
+    gap: 0.5rem;
+    justify-content: center;
+    margin: .5rem;
+    padding: 1rem;
+    width: 50rem;
+    height: auto;
+  }
+  .list-title {
+    padding: 1rem;
+    font-weight: 5px;
+    border-bottom: 1px solid black;
+    margin-bottom: 0.5rem;
+  }
+  .list {
+    background-color: var(--component-white-color);
+    padding: 1rem 0;
+    margin: 2.5rem;
+    width: 15.6rem;
+    max-width: 90%;
+  }
+  .list-style {
+    background-color: var(--component-white-color);
+    padding: 1rem 1rem;
+    padding-left: 0;
+    margin: 2.5rem;
+    width: 20rem;
+    max-width: 100%;
```

Comment:
same interchange values of widths. it will be then become a responsive site

Reply:
Done